bdai_ros2_wrappers/bdai_ros2_wrappers/feeds.py

```diff
@@ -99,3 +101,105 @@ def stream(
     def close(self) -> None:
         """Closes the message feed."""
         self._tape.close()
+
+
+class FramedMessageFeed(MessageFeed):
+    """A message feed decorator, incorporating transforms using a `TransformFilter` instance."""
+
+    def __init__(
+        self,
+        feed: MessageFeed,
+        target_frame_id: str,
+        *,
+        tolerance_sec: float = 1.0,
+        tf_buffer: Optional[tf2_ros.Buffer] = None,
+        history_length: Optional[int] = None,
+        node: Optional[Node] = None,
+    ) -> None:
+        """Initializes the message feed.
+
+        Args:
+            feed: the upstream message feed.
```

Comment:
should we use `@param` to conform with Doxygen formatting?

Reply:
As far as I've seen, we've been using [Google's styleguide](https://google.github.io/styleguide/pyguide.html) for Python docstrings everywhere. We could use Doxygen's, or Numpy's, or Sphinx's instead. I would do that in follow-up PRs though.